packages/gui/src/components/gui/gui.css

```
@@ -238,6 +239,15 @@
     border: 1px solid $looks-secondary;
     border-bottom-left-radius: $radius;
     box-sizing: content-box; /* To match scratch-block vertical toolbox borders */
+    
+    transition: background 0.35s, border 0.35s; 
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)